### ISSUE TITLE
@placeholder no longer a candidate for acc description

### ISSUE THREAD

**Issue by jasonkiss** [stevefaulkner/HTML5accessibility#62]
Update the [`placeholder` attribute test page](https://thepaciellogroup.github.io/AT-browser-tests/test-files/placeholder-att.html) to reflect that `placeholder` no longer participates in the accessible description computation as [outlined in the HTML-AAM](https://rawgit.com/w3c/html-aam/gh-pages/index.html#input-type-text-input-type-password-input-type-search-input-type-tel-input-type-url-and-textarea-element).

**Comment by cookiecrook** [comment]
@stevefaulkner It's challenging to update this since it requires a patch to both the website and the test files which are in another repository. What to you recommend? Updating the results here and filing a new issue to the TPG repo?

As for this repo, the site fix seems to be: mark Safari as passing and mark Edge as failing. But I'd prefer to avoid that political pitfall unless @dstorey gives his blessing on the Edge edit. Otherwise, I'll just mark Safari as passing.

**Comment by cookiecrook** [comment]
#72 Updates Safari results

**Comment by dstorey** [comment]
@jasonkiss @stevefaulkner this was correct but looks like the spec changed after we implemented :(

Is placeholder just removed from accDesc or also accName too? (presumably just the former)

If the former the fix is either to remove ph-002 and ph-003 tests or to leave them to explicitly test against the previously correct behavior and change the test desc to say placeholder shouldn't be the accDesc.

**Comment by dstorey** [comment]
Regarding Edge implementation, if this is the new behaviour I'll work with the team to get Edge behaviour updated for an upcoming release.

**Comment by stevefaulkner** [comment]
@dstorey yeah if you can update would be great! the change you suggest sounds good.

**Comment by dstorey** [comment]
Please see PR at https://github.com/ThePacielloGroup/AT-browser-tests/pull/62
